# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoConduit.cpp
# Commit: 9d8f6e354d8c
# Full Hash: 9d8f6e354d8c332bab7e433252077ec0e1fa8f04
# Author: John Lin <jolin@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - remove MediaCodecVP8Codec. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115531
# ==============================================================================

diff -r 7675ead797c8 -r 9d8f6e354d8c dom/media/webrtc/libwebrtcglue/VideoConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/VideoConduit.cpp	Fri Oct 08 15:57:53 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/VideoConduit.cpp	Thu Oct 07 12:11:40 2021 -0700
@@ -41,10 +41,6 @@
 #  include "VideoEngine.h"
 #endif
 
-#ifdef MOZ_WEBRTC_MEDIACODEC
-#  include "MediaCodecVideoCodec.h"
-#endif
-
 // for ntohs
 #ifdef _MSC_VER
 #  include "Winsock2.h"